### PR TITLE
Update addresses for veSDL contracts deployed on MainNet

### PR DESCRIPTION
**Pull request by nguyenaiden**
***Description***: veSDL related contracts have been deployed on MainNet. This has added those addresses for: 

- VotingEscrow
- GaugeController
- Minter
- FeeDistributor
- GaugeHelperContract

Additionally:
- All uses of `HelperContract` has been converted to `GaugeHelperContract`
- Deleted extraneous MinterAddress in constant file as they were duplicates

### CODE REVIEW

**Review by ug02fast — APPROVED** [review]

File: src/constants/index.ts [hunk 1]
```diff
@@ -188,11 +188,6 @@ export const MINICHEF_CONTRACT_ADDRESSES = buildAddresses({
   [ChainId.EVMOS]: "0x0232e0b6df048c8CC4037c52Bc90cf943c9C8cC6",
 })
 
-export const MINTER_ADDRESSES = buildAddresses({
```
Comment (hammeiam) [inline src/constants/index.ts:191]: oh shit was this a double entry?

File: src/constants/index.ts [hunk 2]
```diff
@@ -558,26 +553,31 @@ export const SDL_TOKEN_ADDRESSES = buildAddresses({
 })
 
 export const VOTING_ESCROW_CONTRACT_ADDRESS = buildAddresses({
-  [ChainId.MAINNET]: "0xb6beB7b30E3e6e041Ee590776C5807B6cf0439CA",
   [ChainId.HARDHAT]: "0x457cCf29090fe5A24c19c1bc95F492168C0EaFdb",
+  [ChainId.MAINNET]: "0xD2751CdBED54B87777E805be36670D7aeAe73bb2",
```
Comment (hammeiam) [inline src/constants/index.ts:557]: ✅

File: src/constants/index.ts [hunk 3]
```diff
@@ -558,26 +553,31 @@ export const SDL_TOKEN_ADDRESSES = buildAddresses({
...
 })
 
 export const FEE_DISTRIBUTOR_ADDRESSES = buildAddresses({
   [ChainId.HARDHAT]: "0x8A93d247134d91e0de6f96547cB0204e5BE8e5D8",
+  [ChainId.MAINNET]: "0xabd040A92d29CDC59837e79651BB2979EA66ce04",
```
Comment (hammeiam) [inline src/constants/index.ts:562]: ✅

File: src/constants/index.ts [hunk 4]
```diff
@@ -558,26 +553,31 @@ export const SDL_TOKEN_ADDRESSES = buildAddresses({
...
 })
+
 export const GAUGE_MINTER_ADDRESSES = buildAddresses({
   [ChainId.HARDHAT]: "0xB82008565FdC7e44609fA118A4a681E92581e680",
+  [ChainId.MAINNET]: "0x358fE82370a1B9aDaE2E3ad69D6cF9e503c96018",
```
Comment (hammeiam) [inline src/constants/index.ts:567]: ✅

File: src/constants/index.ts [hunk 5]
```diff
@@ -558,26 +553,31 @@ export const SDL_TOKEN_ADDRESSES = buildAddresses({
...
 })
 export const SPA_TOKEN_ADDRESSES = buildAddresses({
   [ChainId.ARBITRUM]: "0x5575552988A3A80504bBaeB1311674fCFd40aD4B",
 })
 
-export const HELPER_CONTRACT_ADDRESSES = buildAddresses({
+export const GAUGE_HELPER_CONTRACT_ADDRESSES = buildAddresses({
   [ChainId.HARDHAT]: "0x381445710b5e73d34aF196c53A3D5cDa58EDBf7A",
+  [ChainId.MAINNET]: "0x8020E4134AD6a694AdbE9521a12C751e67CE9861",
```
Comment (hammeiam) [inline src/constants/index.ts:575]: ✅

File: src/constants/index.ts [hunk 6]
```diff
@@ -558,26 +553,31 @@ export const SDL_TOKEN_ADDRESSES = buildAddresses({
...
 })
 
 export const GAUGE_CONTROLLER_ADDRESSES = buildAddresses({
   [ChainId.HARDHAT]: "0x5fc748f1FEb28d7b76fa1c6B07D8ba2d5535177c",
+  [ChainId.MAINNET]: "0x99Cb6c36816dE2131eF2626bb5dEF7E5cc8b9B14",
```
Comment (hammeiam) [inline src/constants/index.ts:580]: ✅

**Review by hammeiam — APPROVED** [review]
Looks good to me, though I think GH is complaining that there's another instance of `Minter` in `TokenClaimDialogue` that needs to be addressed

**Review by ug02fast — APPROVED** [review]

**Pull request merged**